docs: add more information about egress in LGP hybrid (#728)
langchain-ai/docs: src/langgraph-platform/hybrid.mdx

If you would like to deploy to Kubernetes, you can follow the [Hybrid deployment guide](/langgraph-platform/deploy-hybrid).
</Tip>

> ### Egress to LangSmith and the control plane

Reviewer: ```suggestion
### Egress to LangSmith and the control plane

```

In the hybrid deployment model, your self-hosted data plane will send network requests to the control plane to poll for changes that need to be implemented in the data plane. Traces from data plane deployments also get sent to the LangSmith instance integrated with the control plane. This traffic to the control plane is encrypted, over HTTPS. The data plane authenticates with the control plane with a LangSmith API key.

> In order to enable this egress, you may need to update internal Firewall rules or cloud resources such as Security Groups. You can find a list of IP addresses to allow [here](/langsmith/cloud-architecture-and-scalability#ingress-into-langchain-saas).

Reviewer: ```suggestion
In order to enable this egress, you may need to update internal firewall rules or cloud resources (such as Security Groups) to [allow certain IP addresses](/langsmith/cloud-architecture-and-scalability#ingress-into-langchain-saas).
```

<Warning>

> We do not currently support traffic over Private Link or Private Service Connect. This traffic will go over the internet.

Reviewer: ```suggestion
AWS PrivateLink or GCP Private Service Connect is currently not supported. This traffic will go over the internet.
```
